test/functional/wallet_import_rescan.py

```diff
@@ -166,7 +166,7 @@ def run_test(self):
 
         # Create new transactions sending to each address.
         for i, variant in enumerate(IMPORT_VARIANTS):
-            variant.sent_amount = 10 - (2 * i + 1) / 8.0
+            variant.sent_amount = 1 - (2 * i + 1) / 128
```

Comment:
Curious what's prompting these value changes.

Reply:
> Curious what's prompting these value changes.

Change above is This is just reducing